en/V1-IoT_Ecosystem_Requirements.md

```diff
@@ -64,10 +64,10 @@ To ensure the security of all software produced, the build process for the syste
 | -- | ---------------------- | - | - | - |
 | **1.4.1** | Verify that devices can collect logs about events with security implications, such as successful and failed authentication attempts, access to debugging functionality etc. | | ✓ | ✓ |
 | **1.4.2** | Verify that collected logs have adequate granularity to enable actionable insights and alerts. They should include, at a minimum, the type of event, timestamp, source, outcome and identify of involved actors. | | ✓ | ✓ |
```

Comment:
How about the following?

Verify that collected logs have the adequate granularity to enable actionable insights and alerts. Logs should include, at a minimum, the type of event, timestamp, source, outcome, and identification of involved actors.

Reply:
1.4.2 adapted accordingly in e9b9a5ae73d6af99c246e8f5a3903afb493bfb7b  :)